c2cgeoportal/scaffolds/create/deploybranch.mk

```diff
@@ -0,0 +1,5 @@
+INSTANCE_ID = $(GIT_BRANCH)
```

Comment:
Do we need this .mk? Couldn't we directly pass the variables in the target?
If we need this, should it be renamed CONST_deploybranch.mk?